docs/docsource/reference/access_controls/bucket_policies/index.rst

```diff
@@ -0,0 +1,21 @@
+Bucket Policy
+=============
+
+Bucket policies afford you the opportunity to restrict or permit access to
+buckets based on policies owned by the account that owns a bucket. Account users
+can create policies for specific buckets and then grant other accounts access
+permissions to the same bucket (and all objects in it).
+
+User accounts can create a bucket policy for any bucket they own, and can grant
+bucket access to other accounts. Once bucket access is granted, all accounts
+with shared access to the bucket can access all objects in the bucket. Bucket
+policies enable accounts (account root users) to restrict access to specific IP
+addresses and to specific HTTP referrers. 
```

Comment:
We're not supporting using IP addresses and HTTP referrers to control permissions yet :)

Reply:
thanks!

Reply:
done.